src/Illuminate/Foundation/Console/SessionFlushCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Cache;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'session:flush')]
+class SessionFlushCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'session:flush';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Flush all user sessions';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $isConfirmed =  $this->confirm('Do you really wish to run this command?',false);
+
+        if($isConfirmed){
+            $driver = config('session.driver');
+            $method_name = 'clean'.ucfirst($driver);
+            if ( method_exists($this, $method_name) ) {
+                try {
+                    $this->$method_name();
+                    $this->components->info('Session Data Flushed Successfully.');
+                } catch (\Exception $e) {
+                    $this->components->error($e->getMessage());
+                }
+            } else {
+                $this->components->error("Unable to clean the sessions of the driver '{$driver}'.");
+            }
+        }
+        else{
+            $this->components->warn('Command Canceled');
```

Comment:
I think this `else` statement is not necessary, you don't see this with other Artisan commands either

Reply:
replaced with ConfirmableTrait